test/UnbrandedProjects/Platform-OpenAI-TypeSpec/src/Generated/Models/CreateEmbeddingResponse.cs

```diff
@@ -49,6 +49,6 @@ internal CreateEmbeddingResponse(CreateEmbeddingResponseObject @object, string m
         /// <summary> The list of embeddings generated by the model. </summary>
         public IReadOnlyList<Embedding> Data { get; }
         /// <summary> The usage information for the request. </summary>
-        public CreateUsage Usage { get; }
+        public CreateEmbeddingResponseUsage Usage { get; }
```

Comment:
This is an anonymous model: https://github.com/Azure/autorest.csharp/blob/37812dab10bf294ef00614f615e6a8df03f46100/test/UnbrandedProjects/Platform-OpenAI-TypeSpec/embeddings/models.tsp#L33